### PR TITLE
Text utility and list

### PR DESCRIPTION
**Pull request by mimansha-swarup**
ps: live link is not working properly casue few updated stylesheet i haven't added in this pr 
## Text Utility
image

## list
image

### CODE REVIEW

**Comment by Swapnilkota108** [comment]
UI looks good. Aligning the logo and the drawer elements in your documentation will make it look more good in my opinion, and your code looks good too.

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    border-radius: .25rem;
+    width: 100%;
+    box-shadow: var(--Shadow-01);
+}
+.list-item{
+    box-shadow: var(--Shadow-01);
+    background-color: var(--Elevation-01);
+    display: flex;
+    align-items:center ;
+    gap:10%;
```
Comment (himadri2110) [inline components/list/list.css]: On lines 11 and 21, try using the same unit for the property `gap`.  

Rest all looks good!

**Review by himadri2110 — APPROVED** [review]

File: components/list/list.css (hunk 1 above)
Comment (mimansha-swarup) [reply components/list/list.css]: ok noted 👍

**Review by srishti-maurya — APPROVED** [review]
looks good

**Comment by preethasri** [comment]
Looks good and neat

**Pull request merged**